README.md

```diff
@@ -54,4 +54,47 @@ pelfs = PelicanFileSystem(“some-director-url”)
 file1 = PelicanMap(“/namespace/file/1”, pelfs=pelfs)
 file2 = PelicanMap(“/namespace/file/2”, pelfs=pelfs)
 ds = xarray.open_mfdataset([file1,file2], engine='zarr')
-```
\ No newline at end of file
+```
+
+### Specifying Endpoints
+
+The following describes how to specify endpoints to get data from, rather than letting PelicanFS and the director determine the best cache. PelicanFS allows you to specify whether to read directly from the origin (bypassing data staging altogether) or to name a specific cache to stage data into. 
+
+**Note**
+> If both direct reads and a specific cache are set, PelicanFS will use the specified cache and ignore the direct reads setting.
+
+
+#### Enabling Direct Reads
+
+Sometimes you might wish to read data directly from an origin rather than via a cache. To enable this at PelicanFileSystem creation, just pass in `dirReads=True` to the constructor.
+
+```python
+pelfs = PelicanFileSystem("some-federation-url", dirReads=True)
+```
+
+Direct reads can also be turned on an off using the class function `set_direct_reads` like so:
+
+```python
+pelfs.set_direct_reads(True)
+hello_world_from_origin = pelfs.cat('/ospool/uc-shared/public/OSG-Staff/validation/test.txt')
+```
+
+#### Specifying a Cache
+
+If you want to specify a specific cache to stage your data into (as opposed to the highest priority working cache), this can be done by passing in a cache URL during PelicanFileSystem construction via the `specifiedCacheUrl` variable:
+
+```python
+pelfs = PelicanFileSystem("some-federation-url", specifiedCacheUrl="some-cache-url")
```

Comment:
I don't love "specified cache".  How about:

```suggestion
pelfs = PelicanFileSystem("some-federation-url", preferred_caches=["https://cache1.example.com", "https://cache2.example.com"])
```

One lesson from the go client: we often want to _add_ a cache to the front of the list, not replace the list.

I like the fact you don't call it `NEAREST_CACHE` here; I think the go client is wrong in that terminology.